app/controllers/bookingsync_portal/admin/rentals_controller.rb

```diff
@@ -9,7 +9,7 @@ def index
         @visible_rentals = current_account.rentals.visible
         @remote_accounts = current_account.remote_accounts
         @remote_rentals_by_account = current_account.remote_rentals.ordered
-          .includes(:remote_account).group_by(&:remote_account)
```

Comment:
That's true, but the problem is that those controllers are really painful to override. Otherwise this hook would not be needed at all, I could just add what I need in action.